File: 2714-left-and-right-sum-differences/left-and-right-sum-differences.py
class Solution:
    def leftRightDifference(self, nums: List[int]) -> List[int]:
        answer = []
        n = len(nums)
        for i in range(n):
            answer.append(abs(sum(nums[0:i])-sum(nums[i+1:n])))
        return answer

File: 2714-left-and-right-sum-differences/left-and-right-sum-differences.cpp
class Solution {
public:
    vector<int> leftRightDifference(vector<int>& nums) {
        int rightsum = 0, leftsum = 0, length = nums.size();
        vector<int> a;
        for(int i=0; i<length; i++){
            rightsum += nums[i];
        }
        for(int i=0; i<length; i++){
            leftsum += nums[i];
            a.push_back(abs(rightsum-leftsum));
            rightsum -= nums[i];
        }
        return a;
    }
};